metagenomics.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python
+import argparse
+import gzip
+import shutil
+import util.file
+import tools.kraken
+import tools.krona
+import tools.diamond
+__commands__ = []
+
+
+def kraken(inFastq1, inFastq2, db, outReport=None, outReads=None,
+           filterThreshold=None, threads=1):
```

Comment:
is the threads=1 argument extraneous? It doesn't get exposed in argparse, and it doesn't appear to be used later in this method.